src/searches/StationFinder/StationFinder.js

```diff
@@ -0,0 +1,38 @@
+import { transform as transformCoords } from 'ol/proj';
+import React from 'react';
+
+import { setCenter, setZoom } from '../../model/map/actions';
+import Search from '../Search';
+
+class StationFinder extends Search {
+  constructor() {
+    super();
+    this.key = '5cc87b12d7c5370001c1d6551c1d597442444f8f8adc27fefe2f6b93';
+  }
+
+  search(value) {
+    return fetch(`https://api.geops.io/stops/v1/?&q=${value}&key=${this.key}`)
+      .then(data => data.json())
+      .then(featureCollection => featureCollection.features);
```

Comment:
Maybe transform directly the featureCollection to openlayers features.

Reply:
This function runs often, so my idea was to keep this as performant as possible.

Reply:
yes you're right